docs/user/security/secure-saved-objects.asciidoc

```diff
@@ -0,0 +1,90 @@
+[role="xpack"]
+[[xpack-security-secure-saved-objects]]
+=== Secure Saved Objects
+
+In {kib} you usually deal with dashboards, visualizations, alerts, actions, advanced settings, spaces etc. These are so called *Saved Objects* that {kib} stores in a dedicated internal {es} index. If such an object includes any sensitive information, for example a PagerDuty’s integration key or email server credentials used by the alert action, {kib} encrypts it and makes sure it cannot be accidentally leaked or tampered with. Moreover, {kib} records any access attempt to such sensitive information in the audit logs. 
```

Comment:
__note:__ That's the shortest introduction to `Saved Objects` I could come up with, happy to get better ideas! :see_no_evil: